setup.py

```diff
@@ -119,9 +116,13 @@
 # For Python3.4, this is OSX 10.6, but we need Thread Local Support (__thread)
 if 'darwin' in sys.platform and PY3:
   mac_target = sysconfig.get_config_var('MACOSX_DEPLOYMENT_TARGET')
+  # The working computer's mac os x version defines what frameworks are
+  # available to build against.
+  mac_version = platform.mac_ver()[0]
```

Comment:
I got errors building locally against 10.7 (missing standard library header files). If that sounds familiar to you and you have an environment fix s.t. the proper configuration is the builder's responsibility, I'd be more than happy to go back to 10.7.